pom.xml

```diff
@@ -137,6 +137,9 @@
         <jackson.databind.version>2.10.0</jackson.databind.version>
         <kafka.version>2.4.0</kafka.version>
         <janus.version>0.4.0</janus.version>
+        <snappy.version>1.1.7.2</snappy.version>
+        <antlr.version>3.5.2</antlr.version>
+        <joda.version>2.10.1</joda.version>
```

Comment:
Please update to 2.10.5 (otherwise dependabot will anyway) or document why an earlier version is required

Reply:
I found no other dependency to joda in the pom, and no error came from using 2.10.1 when using graph repo. Are you certain about the necessity of using a higher version ?